index.bs

```diff
@@ -2385,19 +2389,66 @@ Note: We do not emit an explicit [=source debug data type=] for "<code>[=destina
 we only emit a [=source debug data type/source-success=] type. For this reason, when both limits are hit, just interpret
 it as "<code>[=destination rate-limit result/hit reporting limit=]</code>" to ensure that the most useful report is sent.
 
-To <dfn>check if an [=attribution source=] exceeds the unexpired destination limit</dfn> given an
-[=attribution source=] |source|, run the following steps:
+An [=attribution rate-limit record=] |a| <dfn for="attribution rate-limit record">is less than</dfn>
+an [=attribution rate-limit record=] |b| if any of the following are true:
+
+* |a|'s [=attribution rate-limit record/scope=] is less than |b|'s [=attribution rate-limit record/scope=].
+* |a|'s [=attribution rate-limit record/scope=] is equal to |b|'s [=attribution rate-limit record/scope=]
+    and |a|'s [=attribution rate-limit record/time=] is less than |b|'s [=attribution rate-limit record/time=].
+* |a|'s [=attribution rate-limit record/scope=] is equal to |b|'s [=attribution rate-limit record/scope=],
+    |a|'s [=attribution rate-limit record/time=] is equal to |b|'s [=attribution rate-limit record/time=],
+    and |a|'s <a href="https://html.spec.whatwg.org/multipage/origin.html#serialization-of-a-site">serialized</a> [=attribution rate-limit record/attribution destination=]
+    is less than |b|'s <a href="https://html.spec.whatwg.org/multipage/origin.html#serialization-of-a-site">serialized</a> [=attribution rate-limit record/attribution destination=].
```

Comment:
I think this would be easier to understand if it were written with early returns, e.g.

```
1. If |a|'s scope is less than |b|'s scope, return true.
2. If |a|'s scope is greater than |b|'s scope, return false.
3. ...
```

Reply:
Updated as suggested, thanks.